CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+### Shadows TODO
```

Comment:
Yeah I think so. All shadow properties like this are inside each `ShadowMap` right now.